pyne/mcnp.py

```diff
@@ -1507,6 +1507,9 @@ def mat_from_inp_line(filename, mat_line, densities='None'):
                     # set metadata
         line_index += 1
         mds_line = linecache.getline(filename, mat_line - line_index)
+        # its possible this line is blank and will fail on the .split operation
```

Comment:
which split fails?  A completely blank line is not valid MCNP input.  Do you have an example that fails?